azext_iot/_params.py

```diff
@@ -560,6 +599,26 @@ def load_arguments(self, _):
             help="Maximum number of elements to return. By default query has no cap.",
         )
 
+    with self.argument_context("iot hub topic-space") as context:
+        context.argument(
+            "topic_name",
+            options_list=["--topic-space-name", "--tsn"],
+            help="Topic space name.",
+        )
+        context.argument(
+            "topic_templates",
+            nargs="*",
+            options_list=["--topic-space-templates", "--templates"],
+            help="List of topic space template paths or a text file containing a list of topic "
```

Comment:
Maybe should say something about, if an input is a file, what the file format should be?